lib/improver/metadata/attributes.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""
+Module defining attributes for IMPROVER blended data and the nowcast
+"""
+
+import re
+import warnings
+
+from improver.utilities.cube_metadata import amend_metadata
+
+
+STANDARD_GRID_TITLE_STRING = "UK 2 km Standard Grid"
+SPOT_TITLE_STRING = "UK Spot Values"
```

Comment:
Why are these titles so UK-centric? We will need to handle both UK and global grids, and IMPRO-1347 does not set out to only handle UK grids.

Reply:
Because that is what is currently in the standard.  Evidently we need to update the standard to include titles for global data.  Can you email me offline about this, and when we have a list of what you think is missing we can discuss it with @bjwheltor so that what we implement now is as complete as it reasonably can be.

Reply:
Having had a quick chat with Bruce, my suggestion is as follows.  For the blended global data, the current code will give it a title eg "MOGREPS-G Temperature Forecast" without a grid descriptor.  For spot data, I have added a more general "Spot Values" descriptor to be appended if the input data does not have "UK" in the original title string.  In both of these cases this avoids putting "UK" into the title of a field that is not restricted to the UK area, and therefore makes the metadata correct for that field.